### PR TITLE
CLOUDP-78654: Missing audit filter on convert

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

_Jira ticket:_ CLOUDP-78654

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

### CODE REVIEW

File: e2e/atlas/helper_test.go [hunk 1]
```diff
@@ -173,7 +173,7 @@ func createProject(projectName string) (string, error) {
 	cmd.Env = os.Environ()
 	resp, err := cmd.CombinedOutput()
 	if err != nil {
-		return "", err
+		return "", fmt.Errorf("%s (%w)", string(resp), err)
```
Comment (gssbzn) [inline e2e/atlas/helper_test.go:176]: drive by debug, a tests was failing here and needed to see why

File: internal/convert/process_config.go [hunk 2]
```diff
@@ -363,6 +363,7 @@ func (p *ProcessConfig) auditLog() *opsmngr.AuditLog {
 		Destination: p.auditLogDestination(),
 		Path:        p.AuditLogPath,
 		Format:      p.AuditLogFormat,
+		Filter:      p.AuditLogFilter,
```
Comment (gssbzn) [inline internal/convert/process_config.go:366]: we missed this, the rest is tests

File: internal/convert/process_config_test.go [hunk 3]
```diff
@@ -116,3 +113,166 @@ func TestNewReplicaSetProcessConfig(t *testing.T) {
 
 	assert.Equal(t, expected, result)
 }
+
+func Test_newConfigRSProcess(t *testing.T) {
```
Comment (gssbzn) [inline internal/convert/process_config_test.go:117]: I added a test of ops manager -> mcli but not one mcli -> ops manager, so now we have both

**Review by threebee — APPROVED** [review]
LGTM!

**Pull request merged**